testdata/simple/5a75f537726934e4b62833ab6d5dca41/tests/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "_id": "62dba0159041307f697e6ccc",
+    "build_id": "5a75f537726934e4b62833ab6d5dca41",
+    "build_name": "MCI_enterprise-rhel_job0",
+    "name": "geo_max:CheckReplOplogs",
+    "command": "",
+    "started": "2022-07-23T07:15:33.967Z",
+    "ended": null,
```

Comment:
If we're only going to write this once when the test is created then we can't ever know when the test ended. I imagine we won't need to know the started time/ended time anyway if we won't need them for fetching/merging logs.

Reply:
I can just ditch started/ended anyway since they aren't as useful for the new algorithm to get all of the "global" logs that occur during a test.

Reply:
Removed started/ended